sdk/examples/client/block/00_block_no_payload.rs

```diff
@@ -10,8 +10,8 @@ use iota_sdk::client::{Client, Result};
 async fn main() -> Result<()> {
     // Take the node URL from command line argument or use one from env as default.
     let node_url = std::env::args().nth(1).unwrap_or_else(|| {
-        // This example uses dotenv, which is not safe for use in production.
-        dotenv::dotenv().ok();
+        // This example uses secrets in environment variables for simplicity which should not be done in production.
```

Comment:
You sometimes have `which` and sometimes have `but`. I think `which` is more grammatically correct

Reply:
yup